src/test/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/controller/AnnualPriceAdjustmentsControllerTest.kt

```diff
@@ -63,6 +63,8 @@ class AnnualPriceAdjustmentsControllerTest(@Autowired private val wac: WebApplic
       .andExpect { model { attribute("contractualYearStart", effectiveYearForDate(effectiveDate).toString()) } }
       .andExpect { model { attribute("contractualYearEnd", (effectiveYearForDate(effectiveDate) + 1).toString()) } }
       .andExpect { model { attributeExists("history") } }
+      // including check for the feedback URL
```

Comment:
There is no actual controller for the feedback URL it is effectively added to all existing controllers so including a simple check here.  In this case there is URL.